pkg/scanners/set_impl.go

```diff
@@ -16,7 +15,16 @@ type setImpl struct {
 	integrations map[string]types.ImageScannerWithDataSource
 }
 
-var registryDependentScanners = set.NewFrozenStringSet("clair", "clairify")
+var (
+	// scannerSortPriority determines the order in which to sort scanners
+	// based on scanner type. Types not included will default to 0 and
+	// be sorted to the top.
+	scannerSortPriority = map[string]int{
+		"clair":     1,
+		"scannerv4": 2,
+		"clairify":  3,
```

Comment:
should we use the `TypeName` fields? I'm not sure, but that may also cause a circular dependency...

Reply:
We already have the `pkg/scanners/types` package, maybe just define the string types there and reference from there? Would be more better IMO than redefining strings here + avoids the circular dep.

Reply:
> should we use the TypeName fields? I'm not sure, but that may also cause a circular dependency...
This does cause a circular dependency (tried to do that initially)

> We already have the pkg/scanners/types package, maybe just define the string types there and reference from there? Would be more better IMO than redefining strings here + avoids the circular dep.

Good call, will do this.